readthedocs/projects/version_handling.py

```diff
@@ -56,9 +56,9 @@ def comparable_version(version_string):
     """
     comparable = parse_version_failsafe(version_string)
     if not comparable:
-        if version_string == LATEST_VERBOSE_NAME:
+        if version_string == STABLE_VERBOSE_NAME:
```

Comment:
I don't think that we should do it like this because -

https://github.com/rtfd/readthedocs.org/blob/93bd03b4a8ed42c3a0b3633f114f21a0eac9ea11/readthedocs/projects/templatetags/projects_tags.py#L13-L20

is using `comparable_version` and changing this means that the order in which versions are rendered everywhere will get disturbed. #5157.